user_manual_de/files/access_webdav.rst

```diff
@@ -425,91 +418,93 @@ Der Zugriff auf Ihre Dateien aus Microsoft Office über WebDAV schlägt fehl.
      Stammzertifizierungsstellen" zu "Zertifikate".
-  #. Mit dem rechten Mausklick auf den Ordner Zertifikate, öffnet sich eine
+  #. Nach einem rechten Mausklick auf den Ordner Zertifikate, öffnet sich eine
      Auswahl. Klicken Sie über "Alle Aufgaben" auf "Importieren".
   #. Wählen Sie nun das zuvor gespeicherte Zertifikat von Ihrem Desktop aus.
   #. Nun wird der Punkt "Alle Zertifikate in folgendem Speicher speichern"
      gewählt und auf das Feld "Durchsuchen" gedrückt.
-  #. In diesem Schritt wird zuerst auf "Physischen Speicher" anzeigen geklickt.
+  #. In diesem Schritt wird zuerst auf "Physischen Speicher anzeigen" geklickt.
      Im oberen Feld wird zu "Vertrauenswürdige Stammzertifizierungsstellen"
-     navigiert und "Lokaler Computer" ausgewählt. Mit dem Klick auf "Weiter"
+     navigiert und "Lokaler Computer" ausgewählt. Mit einem Klick auf "Weiter"
      kann der Import "Fertig gestellt" werden.
-  #. Nachdem die Liste aktualisiert wurde, sollte das Zertifikat ersichtlich sein.
+  #. Nachdem die Liste aktualisiert wurde, sollte das Zertifikat in der Liste
+     angezeigt werden.
   #. Öffnen Sie Ihren Browser und navigieren Sie zu den "Einstellungen" und
-     löschen die Browser Historie/Chronik.
-  #. Wechseln Sie nun zu der Windows Systemeinstellung > "Internetoptionen".
-     Im Register "Inhalt" drücken Sie auf "SSL-Status löschen".
-  #. Schließen und öffnen Sie den Browser für einen ersten Testlauf.
+     bereinigen Sie die Browser Historie/Chronik.
+  #. Wechseln Sie nun zu der Windows Systemeinstellung in "Internetoptionen".
+     Im Reiter "Inhalt" drücken Sie auf "SSL-Status löschen".
+  #. Abschließend öffnen Sie den Browser.
 
 Problem
 ^^^^^^^
 
-Wenn der Upload länger als 30 Minuten mit dem Web Client unter Windows 7 dauert,
-können Sie nicht mehr als 50 MB herunterladen oder große Dateien hochladen.
+Wenn der Upload mit dem Web-Client unter Windows 7 länger als 30 Minuten dauert,
+können Sie nicht mehr als 50 MB herunterladen/hochladen.
 
 Lösung
 ^^^^^^
 
-Bekannte Probleme und deren Lösungen sind in dem Artikel KB2668751_ dokumentiert.
+Bekannte Probleme und deren Lösungen sind im Artikel KB2668751_ dokumentiert.
 
 
 Dateizugriff mit cURL
 ---------------------
 
-Da WebDAV eine Erweiterung von HTTP ist, kann cURL verwendet werden, um
-Dateioperationen zu scriptieren.
+Da WebDAV eine Erweiterung von HTTP cURL ist, können Scriptdateien verwendet
+werden.
 
-So erstellen Sie einen Ordner mit dem aktuellen Datum als Namen:
+Erstellen Sie einen Ordner mit dem aktuellen Datum als Namen:
 
 .. code-block:: bash
 
-	$ curl -u user:pass -X MKCOL "https://beispiel.de/nextcloud/remote.php/dav/files/BENUTZERNAME/$(date '+%d-%b-%Y')"
+	$ curl -u user:pass -X MKCOL "https://example.com/nextcloud/remote.php/dav/files/USERNAME/$(date '+%d-%b-%Y')"
 
-So laden Sie eine Datei ``error.log`` in ein Verzeichnis hoch:
+Laden Sie eine Datei ``error.log`` in ein Ordner hoch:
```

Comment:
`in einen Ordner` / `in den Wurzelordner` (oder wie auch immer Root-Verzeichnis übersetzt wird)